server/api/topics.js

```diff
@@ -73,4 +73,19 @@ router.post("/:topic_id/question", (req, res) => {
 			res.json(500);
 		});
 });
+router.put("/:topic_id/question", function (req, res) {
+	const newQuestion = req.body.question_text;
+	const id = req.params.topic_id;
+	console.log(newQuestion);
+	if(!newQuestion){
+		return res.status(400).send("The question can not be empty");
+	}
+	topicDb
+		.updateQuestion( id, newQuestion)
+		.then(() => res.send("Question updated!"))
```

Comment:
As I have seen that you are sending responses in some different ways. `res.send`, `res.json`, `res.status(200).send`. Let's consolidate the response types

Always when sending returning responses contain a status code
* 200 OK (_on success_)
* 400 Bad request (_on  bad requests_)
* 401 Unauthorised (_on unauthenticated requests_)
... [More here](https://httpstatuses.com/)

And,
* Always response with a JSON response body.
In the case of create, update return the  created/updated entity
```
res.status(200).json(updatedTopic)
```
In the error scenario,
```
res.status(500).json({
  error: "Your error message that describes what went wrong"
})
```